### ISSUE TITLE
GDPR compliance youtube-nocookie breaks rel=0 

### ISSUE THREAD

**Issue by jpwdesigns** [wrav/oembed#53]
When the youtube embed happens and the URL is changed to youtube-nocookie.com it does not apply the rel=0 parameter.

**Comment by jpwdesigns** [comment]
Template code:
```
    {{ block.video.render({
          params: {
              autoplay: 0,
              rel: 0,
              mute: 0,
              loop: 0,
              autopause: 0,
          }
      })
    }}
```

Rendered html:
```
<iframe width="480" height="270" src="https://www.youtube-nocookie.com/embed/XXXXXX?feature=oembed" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe>
```

Expected html:
```
<iframe width="480" height="270" src="https://www.youtube-nocookie.com/embed/XXXXXX?feature=oembed&rel=0" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe>
```

**Comment by reganlawton** [comment]
Hmm thats weird and this DOES work when not GDPR turned on?

**Comment by jpwdesigns** [comment]
I'm not sure. I have yet to see it work correctly

**Comment by reganlawton** [comment]
@jpwdesigns This is resolved in v1.3.9
